public/index.html

```diff
@@ -154,11 +168,6 @@ <h3>Eva Asplund</h3>
 									<h3>Jordan Bramble</h3>
 									<p>Data Engineer, Recidiviz. Machine Learning Engineer, Atrium. Former Data Scientist, USDS.</p>
 								</section>
-								<section>
-									<img class="icon major" src="images/josh.jpg" />
-									<h3>Joshua Essex</h3>
-									<p>Co-founder and Technical Lead, Recidiviz. Architect and Technical Lead, Opower.</p>
-								</section>
```

Comment:
It looks like down on line 96 we have an extra, empty `<section>` element which is making the volunteer table wonky at the bottom.